Mixpanel.xcodeproj/project.pbxproj

```diff
@@ -1026,6 +939,7 @@
 		E11594861CFF1491007F8B4F /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
the internet suggests this removes things that aren't allowed in app extensions at compile time. does that mean that if people want to use app extensions they lose a bunch of functionality in their main app as well? or is this a setting that can be different between their app and the extension?